### DISCUSSION TITLE
The $extends method does not seem to exist in prisma.$transaction.

### DISCUSSION THREAD

**Discussion by 884js** [prisma/prisma#18667] [Q&A] [answered]
The $extends method does not seem to exist in prisma.$transaction.
It seems to exist in TransactionClient.

```
 export type TransactionClient = Omit<Prisma.DefaultPrismaClient, '$connect' | '$disconnect' | '$on' | '$transaction' | '$use'>
```

**Comment by nurul3101** [comment, marked as answer]
Hello @Uki884 👋 

Can you please elaborate on what you are trying to achieve?
Are you using interactive transactions?

Also, are you on the latest version of prisma?

There is a related GitHub Issue
- https://github.com/prisma/prisma/issues/16582

were you running into this?

**Reply by nurul3101** [reply]
Hello @Uki884 👋 
Let us know in case you are still facing this issue.

**Reply by nurul3101** [reply]
It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
